java/src/jmri/jmrit/logixng/tools/swing/ImportLogixFrame.java

```diff
@@ -38,7 +38,8 @@
     private JButton _cancelDone;
 
     private static final String SYSLOGIX = InstanceManager.getDefault(LogixManager.class).getSystemNamePrefix() + ":SYS";
-    private static final String INITLOGIX = jmri.jmrit.beantable.LRouteTableAction.getLogixInitializer();
+    private static final String RTXLOGIX = "RTX";
+    private static final String USSLOGIX = "USS CTC:OsIndicator";
```

Comment:
If I understand it correctly, an RTX Logix/Conditional always starts with system name prefix and `:RTX`.
And that an USS Logix/Conditional always starts with system name prefix and `:USS CTC:OsIndicator:`.

I suggest that you do the same with RTX and USSLOGIX as you do with SYSLOGIX and then uses `logix.getSystemName().startsWith(RTXLOGIX)`.

The problem now is that a LogixNG like `IXDanielRTX` is treated as a system logix.

Reply:
The problem is that RTX Logixes (**R**oute **T**o logi**X**) can be IX:RTX, IXRTX or any other historical portion after IX.  Logix started using the system name prefix only recently.  In the past it was fixed at **IX**.  This mean that the system name prefix is not predictable.

Reply:
I see. Thanks for the explanation.